### PR TITLE
Suggested fix for issue #19286

### PR DESCRIPTION
**Pull request by fgm**
The patch corrects the bug and also fixed part of #18676.
- tests/bug_19286.phpt test added to catch the bug and demonstrate its fix
- class initializes nodes list when adding subgraph or cluster
- file converted to UTF-8
- removed terminal ?>

### CODE REVIEW

File: Image/GraphViz.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  *
  * PHP version 4 and 5
  *
- * Copyright (c) 2001-2007, Dr. Volker Göbbels <vmg@arachnion.de> and
+ * Copyright (c) 2001-2007, Dr. Volker GÃ¶bbels <vmg@arachnion.de> and
```
Comment (CloCkWeRX) [inline Image/GraphViz.php:10]: Bit of an encoding problem here :S Can you check your editor's settings?

**Comment by CloCkWeRX** [comment]
Thanks for the pull request. I'll need to push this onto the svn version first, and let it mirror eventually - or shift the package fully to github.

**Comment by fgm** [comment]
The file is now in UTF8. It seems other files in the package are still in ISO-8859-1 or Win-1252.

It certainly looks PEAR SVN is slowly bein abandoned for Github, so it could make sense to follow the movement and complete the move for this project.

**Comment by CloCkWeRX** [comment]
Applied, should end up mirrored shortly.

**Pull request closed without merging**